### PR TITLE
[12.0][ADD] survey_description module

### PR DESCRIPTION
**Pull request by remi-filament**
Add description fields to surveys on backend (these fields are already present on the code but not displayed on backend)
[Odoo PR 52329](https://github.com/odoo/odoo/pull/52329) has been create to include this in Odoo code but since survey was refactored in V13, this may not be merged ever in Odoo code...

### CODE REVIEW

**Review by apollet — APPROVED** [review]

File: survey_description/readme/DESCRIPTION.rst [hunk 1]
```diff
@@ -0,0 +1,10 @@
+This module displays the description fields on backend for :
+* surveys
+* pages
+* questions
+
+and thank you field for surveys.
+
+Note: these fields are present in code, displayed on front-end for survey (when defined) but not displayed on back-end for user to fill.
```
Comment (skeller1) [inline survey_description/readme/DESCRIPTION.rst]: back-end => backend

**Review by skeller1 — CHANGES_REQUESTED** [review]
LGTM :+1:
simple typo in docs (see review).
Also tested it with website_survey addon installed.

File: survey_description/readme/DESCRIPTION.rst (hunk 1 above)
Comment (remi-filament) [reply survey_description/readme/DESCRIPTION.rst]: Thank you Stephan, I just fixed it !

**Review by skeller1 — APPROVED** [review]

**Review by Yoshiro009 — APPROVED** [review]

**Pull request merged**